css/styles.css

```diff
@@ -1,85 +1,138 @@
 * {
-  border-style: border-box;
+  box-sizing: border-box;
   margin: 0;
   padding: 0;
   font-family: 'Open Sans', sans-serif;
   outline: none;
 }
 
 body {
-  background-color: lightgray;
+  background-color: black;
   color: #ffffff;
-  background-image: url(../images/vegas.jpg);
-/*  background-repeat: no-repeat;*/
+}
 
+.section__main-screen {
+  background-image: url(https://www.holidayguru.ie/wp-content/uploads/2016/04/Las-Vegas-Strip-at-night-high-vantage-iStock_000082066557_Large-2.jpg);
```

Comment:
I think it would be good to download this image and store it in our repo. That way we won't have to rely on a third party to host it.